fastlane/Fastfile

```diff
@@ -714,6 +714,9 @@ lane :upload_file_to_s3 do |options|
     s3_bucket = ENV['AWS_BUCKET_NAME']
     s3_folder = ''
 
+    version_number = ''
```

Comment:
is this intended?

Reply:
Yes, those are used to expose values for `send_message_to_mattermost`.